### ISSUE TITLE
How to sort list by date 

### ISSUE THREAD

**Issue by superwebarmy** [ivehement/saf#18]
Hi, I was wondering if we can sort the list of strings by their creation time or modified time. I'm making the use of cache method which returns list of string but it returns in the random order

**Comment by AMuellerAtAHS** [comment]
No, actually you can **not** sort the list of strings by time. Because the strings are only strings and have no information about creation time, modified time or anything else.

You are only able to sort the list with string comparison.

```dart
sortedPaths = paths.sort((a,b) => a.compareTo(b));
```

**Comment by jvoltci** [comment]
In 2.0.0, `list()` returns `SafDocumentFile`s with `lastModified` (epoch ms), so sort in Dart:

```dart
final files = await Saf().list(dir.uri)
  ..sort((a, b) => b.lastModified.compareTo(a.lastModified)); // newest first
```
